src/de/jost_net/JVerein/server/DDLTool/Updates/Update0447.java

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without 
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See 
+ * the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program. If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ **********************************************************************/
+package de.jost_net.JVerein.server.DDLTool.Updates;
+
+import de.jost_net.JVerein.server.DDLTool.AbstractDDLUpdate;
+import de.jost_net.JVerein.server.DDLTool.Column;
+import de.willuhn.util.ApplicationException;
+import de.willuhn.util.ProgressMonitor;
+
+import java.sql.Connection;
+
+public class Update0447 extends AbstractDDLUpdate
+{
+  public Update0447(String driver, ProgressMonitor monitor, Connection conn)
+  {
+    super(driver, monitor, conn);
+  }
+
+  @Override
+  public void run() throws ApplicationException
+  {
+    execute(alterColumn("mail",
+        new Column("versand", COLTYPE.TIMESTAMP, 0, "NULL", false, false)));
+
+    execute(alterColumn("mailempfaenger",
+        new Column("versand", COLTYPE.TIMESTAMP, 0, "NULL", false, false)));
+
+    execute(dropForeignKey("fkAnfangsbestand1", "anfangsbestand"));
+
```

Comment:
Hier gibt es eventuell das gleiche Problem welches ich mit Update0443 hatte.
Bis zur Übergabe von #225 war ein Fehler in AbstractDDLUpdate der dazu führte, dass bei MySQL nicht die spezifizierten ForeignKey Namen benutzt wurden wie bei H2DB sondern automatisch generierte Namen. Das führte dazu, dass die dropForeignKey zu Exceptions führten.
Wurde die MySql Datenbank also vor 225 erzeugt muss man schauen welche Namen sie da hatten und diese löschen. Später erzeugte Datenbanken haben die gleichen Namen wie bei H2DB. Meine Lösung in Update0443 war dann erst ein Try mit den neuen Namen und bei Exception die alten Namen zu nutzen.

Ob das hier auch zutrifft musst du ausprobieren.

Reply:
das muss ich mir nochmal genauer anschauen, ob das hier auch so ist und wenn ja wie ich das lösen muss.

Reply:
Hier ist keine Änderung nötig, da der ForeignKey nicht durch die Methode erstellt wurde sondern durch JVereinUpdateProvider